go/libraries/doltcore/sqle/tables.go

```diff
@@ -98,10 +99,16 @@ func (ds *DoltTableStatistics) NullCount() uint64 {
 }
 
 func (ds *DoltTableStatistics) Histogram(colName string) (*sql.Histogram, error) {
```

Comment:
why do we need `DoltTableStatistics`? just to wrap `map[string]sql.TableStatistics`?

Reply:
it implements `sql.TableStatistics`